### PR TITLE
spec/03_number_spec.rb : Fix comment in Assignment

### PR DESCRIPTION
**Pull request by Uaitt**
The lesson  talks about the use of the method subject and the differences between subject and let. The comment in question tells the user to create a variable (an helper method actually) with let, while I believe it should be created with subject, because: 

1) subject is the topic of the lesson;
2) the variable we are creating is the subject of all the tests, hence it makes more sense to create it with subject instead of let.

Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `String spec: Update instructions for clarity`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [ ] I have ensured any exercise files included in this PR have passed all of their tests

<hr>

**1. Because:**

Closes #XXXXX

**2. This PR:**

**3. Additional Information:**

### CODE REVIEW

**Review by rlmoser99 — CHANGES_REQUESTED** [review]
This change would also need changes in the corresponding spec_answer file.

**Review by Uaitt — COMMENTED** [review]
Done.

File: spec_answers/03_number_answer.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 describe Array do
   context 'when my_array has perfect values' do
-    # Write a let variable 'my_array' that passes all tests.
+    # Write a subject variable 'my_array' that passes all tests.
     let(:my_array) { [-7, 42, 101, 98] }
```
Comment (rlmoser99) [inline spec_answers/03_number_answer.rb:6]:
```suggestion
    subject(:my_array) { [-7, 42, 101, 98] }
```

**Review by rlmoser99 — CHANGES_REQUESTED** [review]
Thanks for updating this PR so quickly. We need 1 more change in the file in spec_answer.

File: spec_answers/03_number_answer.rb (hunk 1 above)
Comment (Uaitt) [reply spec_answers/03_number_answer.rb]: Fixed! Sorry for the inconvenience.

**Review by rlmoser99 — APPROVED** [review]
Thank you for taking the time to made & update this PR.

**Pull request merged**